docs/build/tutorials/nodes-and-staking/run-avalanche-node.md

```diff
@@ -51,28 +51,31 @@ If you want to build the node from source, you're first going to need to install
 
 Run `go version`. **It should be 1.17.9 or above.** Run `echo $GOPATH`. **It should not be empty.**
 
-Download the AvalancheGo repository:
+Download the AvalancheGo repository into your `$GOPATH`:
```

Comment:
I think including `$GOPATH` here will confuse people. I'd just keep it as clone.